libs/clustermatch/settings.py

```diff
@@ -0,0 +1,44 @@
+"""
+General settings. This file is intended to be modified by the user. Each entry
+also provides an alternative way to specify its value using an environment
+variable.
+"""
+
+# Instead of changing this file, you can also use the environment variable name
+# specified for each entry (environment variables supersede these settings).
+
+# Specifies the main directory where all data and results generated are stored.
+# When setting up the environment for the first time, input data will be
+# automatically downloaded into a subfolder of ROOT_DIR.
+#
+# Default: if not specified (None), it defaults to the 'cm_gene_expr' subfolder
+# in the temporary directory of the operating system (i.e. '/tmp/cm_gene_expr'
+# in Unix systems).
+# Environment variable: CM_ROOT_DIR
+ROOT_DIR = None
+
+# Specifies the directory where the manuscript git repository was
+# cloned/downloaded. If None, manuscript figures and other related files will
+# not be generated.
```

Comment:
So will this save the figures directly to the manuscript directory? That's a neat idea - I always just save them locally and copy them over by hand, but this makes sure they're always in sync.

Reply:
Yes, directly to the manuscript folder!